libs/components/src/next/table/vl-table.component.cy.ts

```diff
@@ -0,0 +1,626 @@
+            });
+    });
+
+    it('should contain a table with columns', () => {
+        mountDefault({});
+
+        cy.get('vl-table-next').find('table').should('have.class', 'vl-table-next');
+        cy.get('vl-table-next')
+            .find('table')
+            .find('tbody')
+            .children()
+            .each((row) => {
+                if (!row.attr('data-details-id')) {
+                    cy.wrap(row)
+                        .children()
+                        .each((cell, cellIndex) => {
+                            if (!cell.children('vl-button-next').length) {
+                                cy.wrap(cell).should('contain.text', 'Entry line ' + (cellIndex + 1));
+                            }
+                        });
+                }
+            });
+    });
+
+    it('should contain a table with hover styling', () => {
+        mountDefault({ hover: true });
+
+        cy.get('vl-table-next')
+            .find('table')
+            .should('have.class', 'vl-table-next')
+            .should('have.class', 'vl-table-next--hover');
+    });
+
+    it('should contain a table with a matrix', () => {
+        mountDefault({ matrix: true });
+
+        cy.get('vl-table-next')
+            .find('table')
+            .should('have.class', 'vl-table-next')
+            .should('have.class', 'vl-table-next--matrix');
+    });
+
+    it('should contain a table with a grid', () => {
+        mountDefault({ grid: true });
+
+        cy.get('vl-table-next')
+            .find('table')
+            .should('have.class', 'vl-table-next')
+            .should('have.class', 'vl-table-next--grid');
+    });
+
+    it('should contain a table with a zebra grid', () => {
+        mountDefault({ zebra: true });
+
+        cy.get('vl-table-next')
+            .find('table')
+            .should('have.class', 'vl-table-next')
+            .should('have.class', 'vl-table-next--zebra');
+    });
+
+    it('should contain a table that collapsed on the medium breakpoint', () => {
+        mountDefault({ collapsedM: true });
+
+        cy.get('vl-table-next')
+            .find('table')
+            .should('have.class', 'vl-table-next')
+            .should('have.class', 'vl-table-next--collapsed-m');
+    });
+
+    it('should contain a table that collapsed on the small breakpoint', () => {
+        mountDefault({ collapsedS: true });
+
+        cy.get('vl-table-next')
+            .find('table')
+            .should('have.class', 'vl-table-next')
+            .should('have.class', 'vl-table-next--collapsed-s');
+    });
+
+    it('should contain a table that collapsed on the extra small breakpoint', () => {
+        mountDefault({ collapsedXS: true });
+
+        cy.get('vl-table-next')
+            .find('table')
+            .should('have.class', 'vl-table-next')
+            .should('have.class', 'vl-table-next--collapsed-xs');
+    });
+});
+
+const mountExpandable = ({
```

Comment:
normaal niet, maar in dit geval misschien de te mounten code en utility methodes in een apart bestand ernaast zetten?